certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -1303,7 +1313,13 @@
       "description": "The id of the relationship. The id is unique within the digital twin and case sensitive.",
       "x-ms-summary": "Relationship id"
     },
-    "addValueTemplate":{
+    "continuationToken": {
+      "name": "continuationToken",
+      "in": "query",
+      "required": false,
+      "type": "string"
```

Comment:
@wuqiten The 'x-ms-summary' property is required, and the 'description' property is required. Please add both.